docs/searching-algorithms/searching-algorithms.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: What are Search Algorithms?
+slug: /searching-algorithms
+---
+
+## Introduction
+
+Searching is a method of finding whether an element or a value exists in the given list of elements or values. If the element exists, then it returns its position. In the real world, there are many useful applications based on searching, be that seeking a whole database to find a certain file or it might be a simple connection from your contact list. If you wanna find something from a collection of items, you gotta need this algorithmic process.
```

Comment:
Few changes:
- `If you want to find something....`
- `you got to need this algorithmic process.`